chroma-bundles/include/find_deps.py

```diff
@@ -92,9 +92,12 @@ def _find_deps(self, pkg, excludes):
 
     def disable_all_repos(self):
         for repo in self.yb.repos.findRepos("*"):
+            # these are hardcoded to the distro defaults and our own
+            # infrastructure right now
+            # these probably need to be externally specified
             if repo.id != "base" and repo.id != "core-0" and \
```

Comment:
Syntax would probably be nicer with repo.id not in [...] , definitely not worth repush